libraries/matchers.rb

```diff
@@ -4,7 +4,7 @@
     append_if_no_line: [:edit],
     delete_from_list: [:edit],
     delete_lines: [:edit],
-    replace_or_add: [:edit],
+    replace_or_add: [:edit]
```

Comment:
Upgraded to a new version of chef-dk.  There was also a cook style change that added the last comma.